tardis/interfaces/siteadapter.py

```diff
@@ -113,6 +117,41 @@ def handle_response(
 
         return translated_response
 
+    def drone_environment(
+        self, drone_uuid: str, meta_data_translation_mapping: dict = None
+    ):
+        """
+        Method to get the drone environment to be exported to batch jobs
+        providing the actual resources in the overlay batch system. It
+        translates units of drone meta data into a format the overlay
+        batch system is expecting. In addition, the drone_uuid is adding to
+        allow for matching drones to actual resources provided in the overlay
+        batch system.
+        :param drone_uuid: The unique id which is assigned to every drone on creation
+        :type drone_uuid: str
+        :param meta_data_translation_mapping: Mapping used for the meta data translation
+        :type meta_data_translation_mapping: dict
+        :return: Translated
+        :rtype: dict
+        """
+        meta_data_translation_mapping = meta_data_translation_mapping or {
+            "Cores": 1,
+            "Memory": 1024,
+            "Disk": 1024,
+        }  # defaults to units expected by HTCondor
```

Comment:
Can we make the default configurable easily? Assuming there is only one OBS per TARDIS instance, storing this globally would suffice.

Reply:
The assumption is correct, `TARDIS` supports only one OBS at a time. Thinking about it, I am actual not sure if the SLURM OBS supports something similar to the ClassAd mechanism in HTCondor, so it is unclear if we can make use of such environment variables at all. 
The way to go is to add a property to the `BatchSystemAdapter` containing the translation map for the OBS. The only way the `BatchSystemAdapter` can communicate to the `SiteAdapter`is via the `Drone`. So, the `SiteAdapter` has to ask the drone to get the defaults for the OBS. 
@maxfischer2781: Does this sounds reasonable to you?

Reply:
Probably, I was a bit to fast. Unfortunately, the `SiteAdapter` has no access to the `Drone` yet. :-/

Reply:
We have to pass it as a parameter to the https://github.com/MatterMiners/tardis/blob/0ff3b41dd97e308da6e64aa5bf493396601b512c/tardis/resources/dronestates.py#L78 call.

Reply:
Honestly, I do not think it is worth the effort since there is currently no need for this feature. Do you agree, that I add a issue describing the way to go and remove the configurable for now?

Reply:
As agreed in the meeting, we are going to implement the "way to go" as described above.

Reply:
The requested changes have been implemented.